apps/web/pages/api/link.ts

```diff
@@ -39,7 +39,10 @@ async function handler(req: NextApiRequest, res: NextApiResponse<Response>) {
         username: "" /* Not used in this context */,
         role: UserPermissionRole.USER,
       },
-      hasValidLicense: true,
+      license: {
+        valid: true,
+        key: "",
+      },
```

Comment:
License validity has been moved to its own object together with the key in order to have it at hand for other usages.

Reply:
I needed the key in the telemetry context, perhaps we can remove it from telemetry so I can prescind from including it in the session, which I don't really like